File: app/src/main/java/com/github/quentin7b/sn/widget/StickyWidgetService.java
package com.github.quentin7b.sn.widget;

import android.content.Intent;
import android.widget.RemoteViewsService;

import java.sql.SQLException;

/**
 */
public class StickyWidgetService extends RemoteViewsService {

    @Override
    public RemoteViewsFactory onGetViewFactory(Intent intent) {
        return new StickyWidgetRemoteViewsFactory(getApplicationContext());
    }
}

File: app/src/main/java/com/github/quentin7b/sn/widget/StickyWidgetService.kt
package com.github.quentin7b.sn.widget

import android.content.Intent
import android.widget.RemoteViewsService

import java.sql.SQLException

class StickyWidgetService : RemoteViewsService() {

    override fun onGetViewFactory(intent: Intent): RemoteViewsService.RemoteViewsFactory {
        return StickyWidgetRemoteViewsFactory(applicationContext)
    }
}